apis/instantlaunches.go

```diff
@@ -50,7 +54,9 @@ func (i *InstantLaunchesAPI) PullItems(ctx context.Context) ([]map[string]interf
 
 	resp, err := httpClient.Do(req)
 	if err != nil {
-		resp.Body.Close()
+		if resp != nil {
+			resp.Body.Close()
+		}
```

Comment:
This was causing a panic because my local config had a bad setting for the app-exposer URL. So I fixed the config but also figured I'd fix the bug.